WORKSPACE

```diff
@@ -129,3 +129,7 @@ rbe_autoconfig(
     # a specific Linux kernel that comes with "libx11" in order to run headless browser tests.
     repository = "google/rbe-ubuntu16-04-webtest",
 )
+
+load("@npm_angular_dev_infra_private//benchmark/browsers:browser_repositories.bzl", "browser_repositories")
+
+browser_repositories()
```

Comment:
Add a comment that explains what this is setting up?

Reply:
Can you also add a TODO for us deduping the browsers? With this change, we'll have two versions of chromium in the repository. We probably should use `chromium` from the dev-infra package (as that one has RBE improvements)